app/controllers/groups_controller.rb

```diff
@@ -4,29 +4,35 @@ class GroupsController < ApplicationController
 
   def index
     @groups = Group.all
+    redirect_to teacher_groups_url(@user)
```

Comment:
Is it ok for this controller to be completely related to teachers?
Will you have a groups controller to manage members?

Reply:
Yes it is. The only with access to manage groups is the teacher and all his student must be assigned to some group.